src/Gallery/Images.tsx

```diff
@@ -21,8 +21,8 @@ export const Images: FC<GalleryTab> = ({ items, upload, setSelected, selectedId
 
   return (
     <section className="whppt-gallery whppt-gallery__main-container">
-      <div className="whppt-gallery__grid">
-        <WhpptImageUploader uploadImage={uploadImage} />
+      <div className="whppt-gallery-grid whppt-gallery-grid--images">
```

Comment:
Looking at the diff now on the PR I think `whppt-gallery-grid--images` is incorrect (is not a modifier), should be `__images`. Will change after feedback.

Reply:
agreed